### ISSUE TITLE
Error backing up site with clone plugin. Error restoring site from backup plugin.

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#7647]
**Describe the bug**
See previous report re>Clone plugin only works one time, but also produced the same result as below. In that config, site.old had the `$global['mysqli_charset'] = 'latin1'; in its configuration.php, but site.new did not.

I've added the 'latin1' config to site.new now.

Using Backup, I've exported db/videos from site.old, and restored them on site.new using same.

On site.new, there is a strange \ occurring before all ' characters, and other weirdness. _If I go to edit these and save, every time I do an additional backslash is added.
![Screen Shot 2023-02-08 at 8 42 50 PM](https://user-images.githubusercontent.com/11711050/217719800-07de029c-87f4-44be-918e-bc23e5154993.png)

**To Reproduce**
1. Clonesite or Backup
2. Site.old is running 
`Ubuntu 22.04.1 LTS mysql  Ver 8.0.31-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))`
3. Site.new is running
`Ubuntu 22.04.1 LTS mysql  Ver 8.0.32-0ubuntu0.22.04.2 for Linux on x86_64 ((Ubuntu))`

TL;dr export from site running mysql v 8.0.31 seems like this might be an issue when importing to mysql v 8.0.32 sites.

![Screen Shot 2023-02-08 at 8 39 10 PM](https://user-images.githubusercontent.com/11711050/217719513-d7274050-1822-4474-a533-00b173f55a8e.png)

--------

UPDATE: after updating site.old, we have the same version of mysql between both boxes.

From site.old exported the database, copied to site.new, and after installing - same issue persists with back slashes.

Note that other site features are not working, like remove branding->upload image for whatever reason is not working - that may be unrelated - wasn't a problem before.

**Comment by DanielnetoDotCom** [comment]
I saw something similar in the past, please try to run this to fix

https://github.com/WWBN/AVideo/blob/master/install/fixSlashN.php

```
cd /var/www/html/AVideo/install
php fixSlashN.php
```
